commons/src/main/java/io/hyscale/commons/io/StructuredOutputHandler.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.io;
+
+import com.google.gson.*;
+import io.hyscale.commons.constants.ToolConstants;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class StructuredOutputHandler {
+    public static final Gson GSON_BUILDER = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+
+    private List<String> errorMessages = new ArrayList<>();
+
+    public void addErrorMessage(String message, String... args) {
+        if (!StringUtils.isEmpty(message)) {
```

Comment:
Instead of keeping everything inside if, you can return if message is empty
```
if(StringUtils.isEmpty(message)){
return;
}
// Continue code
```